stable/datadog/README.md

```diff
@@ -1,177 +1,131 @@
 
 ### Enabling the Datadog Cluster Agent
 
 Read about the Datadog Cluster Agent in the [official documentation](https://docs.datadoghq.com/agent/kubernetes/cluster/).
 
-Run the following if you want to deploy the chart with the Datadog Cluster Agent.
-Note that specifying `clusterAgent.metricsProvider.enabled=true` will enable the External Metrics Server.
-If you want to learn to use this feature, you can check out this [walkthrough](https://github.com/DataDog/datadog-agent/blob/master/docs/cluster-agent/CUSTOM_METRICS_SERVER.md).
-The Leader Election is enabled by default in the chart for the Cluster Agent. Only the Cluster Agent(s) participate in the election, in case you have several replicas configured (using `clusterAgent.replicas`.
-You can specify the token used to secure the communication between the Cluster Agent(s)q and the Agents with `clusterAgent.token`. If not specified, a random one will be generated and you will be prompted a warning when installing the chart.
+Run the following if you want to deploy the chart with the Datadog Cluster Agent: 
 
 ```bash
 helm install --name datadog-monitoring \
-    --set datadog.apiKey=YOUR-API-KEY-HERE \
-    --set datadog.appKey=YOUR-APP-KEY-HERE \
+    --set datadog.apiKey=<DATADOG_API_KEY> \
+    --set datadog.appKey=<DATADOG_APP_KEY \
     --set clusterAgent.enabled=true \
     --set clusterAgent.metricsProvider.enabled=true \
     stable/datadog
 ```
 
-## Uninstalling the Chart
+**Note**: Specifying `clusterAgent.metricsProvider.enabled=true` enables the External Metrics Server.
+If you want to learn to use this feature, you can check out this [Datadog Cluster Agent walkthrough](https://github.com/DataDog/datadog-agent/blob/master/docs/cluster-agent/CUSTOM_METRICS_SERVER.md).
+
+The Leader Election is enabled by default in the chart for the Cluster Agent. Only the Cluster Agent(s) participate in the election, in case you have several replicas configured (using `clusterAgent.replicas`.
+
+You can specify the token used to secure the communication between the Cluster Agent(s)q and the Agents with `clusterAgent.token`. If not specified, a random one is generated and a warning is prompted when installing the chart.
+
+### Updating
+#### From < 1.19.0 to >= 1.19.0
+
+Version `1.19.0` introduces the use of release name as full name if it contains the chart name(`datadog` in this case).
+E.g. with a release name of `datadog`, this renames the `DaemonSet` from `datadog-datadog` to `datadog`.
+The suggested approach is to delete the release and reinstall it.
 
-To uninstall/delete the `my-release` deployment:
+### Uninstalling the Chart
+
+To uninstall/delete the `<RELEASE_NAME>` deployment:
 
 ```bash
-helm delete my-release
+helm delete <RELEASE_NAME>
 ```
 
 The command removes all the Kubernetes components associated with the chart and deletes the release.
```

Comment:
Actually, to remove the release, one must use `--purge`. If `--purge` is not specified, the chart name cannot be reused in a new install